pyvista/core/filters/data_set.py

```diff
@@ -2004,23 +1996,43 @@ def glyph(
                 )
                 orient = False
 
+        set_active_scalars_vectors = False
```

Comment:
I was confused by the naming here at first.  It isn't whether active scalars or vectors or set, but whether active scalars or vectors need to be reset?

Reply:
Yes, exactly. `set_active_scalar_vectors_again`? Not sure how to make this clear.

Reply:
What about moving this line to be next to `source_data=dataset` and changing name to `set_actives_source_data`?

Reply:
Good suggestion, I made the change